modules/statics/src/coins.ts

```diff
@@ -588,10 +588,9 @@ export const coins = CoinMap.fromCoins([
     '',
     'TST'
   ),
-  ofcerc20('ofcerc', 'ERC Token', 0, UnderlyingAsset.ERC),
   ofcerc20('ofcusdt', 'Tether', 6, UnderlyingAsset.USDT),
   ofcerc20('ofcusdc', 'USD Coin', 6, UnderlyingAsset.USDC),
-  tofcerc20('ofcterc', 'ERC Test Token', 0, UnderlyingAsset.ERC),
+  tofcerc20('ofcterc', 'Test ERC Token', 0, UnderlyingAsset.TERC),
   tofcerc20('ofctusdt', 'Test Tether', 6, UnderlyingAsset.USDT),
   tofcerc20('ofctusdc', 'Test USD Coin', 6, UnderlyingAsset.USDC),
```

Comment:
as per our conversation in slack these basically dont exist I think? I've been directed to use TERC instead which suggests this actually doesnt exist on the ropstine network

 @marktoda

Reply:
> ropstine network.


We use kovan tesnet not ropsten